Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -75,7 +153,21 @@ public static int[] InsertFirst(int[] array, int number)
         /// <returns>A new array with element added in the end of array. If an array is empty or null, returns new array with number in it.</returns>
         public static int[] InsertLast(int[] array, int number)
         {
-            // ToDo: implement.
+            if (array == null)
+            {
+                int[] lengthenedArray = new int[1] { number };
+                array = lengthenedArray;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Introduce a new variable instead of reusing the parameter 'array'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4887772)